README.md

```diff
@@ -93,7 +93,7 @@ Classics are common patterns that return in Round The Table matches, noticably b
 - The player saving the other will loose its life if the player makes a mistake. This way there is always a risk involved helping others.
 
 ### Promoting Diversity
-Diversity at HackerOne is a hot topic right now. We aim to be as diverse as possible. Not only when working on world-class product, but also during the teambuilding. Therefore any diverse playing member (women, foreigners, visitors from SF) will start any round with one life. That excludes players who own Frisian passport, and players from Brabant.
+Diversity at HackerOne is a hot topic. We aim to be as diverse as possible. Not only when working on a world-class product, but also during teambuilding. Therefore any diverse playing member (women, foreigners, visitors from SF) will start any round with one life. That excludes players who own Frisian passport and players from Brabant.
```

Comment:
In comes reputation! I would suggest we add +7 points in the overall scoring for won matches, -5 for pulling a jeurissen, etc. @mvgijssel probably has some super strong opinions on what gem to use to implement reputation.